packages/main/src/Table.hbs

```diff
@@ -6,6 +6,15 @@
 	<table border="0" cellspacing="0" cellpadding="0" @keydown="{{_onkeydown}}" role="table">
 		<thead>
 			<tr id="{{_columnHeader.id}}" role="row" class="ui5-table-header-row" tabindex="{{_columnHeader._tabIndex}}" style="height: 48px" @click="{{_onColumnHeaderClick}}">
+				{{#if isMultiSelect}}
+					<th class="ui5-table-select-all-column">
+						<ui5-checkbox 
+							@change="{{_selectAll}}"
```

Comment:
Perhaps we should select the "selectAll" checkbox, when all rows are selected

Reply:
Now, it works in similar way